demos/job_dvid_ffn.yml

```diff
@@ -0,0 +1,33 @@
+# DVID:
+host_name: 3.209.156.251:8001
+uuid: 56e2e4251774426abdde8cdee4be747e
```

Comment:
Is this a public token?

Reply:
yeah its the public identifier for their "channel"